### PR TITLE
refactoring smoke test

### PR DESCRIPTION
**Pull request by jitendar-singh**
- Cleaning up the resources before running the jenkins-ephemeral.feature as it was causing the test scenario to fail
- Removed the plugin feature file and steps as its not yet implemented
- Updated the README.Md

### CODE REVIEW

File: smoke/features/jenkins-ephemeral.feature [hunk 1]
```diff
@@ -5,6 +5,15 @@ Feature: Deploy Jenkins on openshift using template based install
...
     Given Project [TEST_NAMESPACE] is used
+    Then we delete deploymentconfig.apps.openshift.io "jenkins"
+        And  we delete route.route.openshift.io "jenkins"
+        And  delete configmap "jenkins-trusted-ca-bundle"
+        And  delete serviceaccount "jenkins"
+        And  delete rolebinding.authorization.openshift.io "jenkins_edit"
+        And  delete service "jenkins-jnlp"
+        And  delete service "jenkins"
+        And delete all deploymentconfig
+        And delete all remaining test resources
```
Comment (akram) [inline smoke/features/jenkins-ephemeral.feature:16]: What about creating a single @step that does that in one shot?

**Review by akram — COMMENTED** [review]
Except my single comment, that looks good to me.

File: smoke/features/jenkins-ephemeral.feature (hunk 1 above)
Comment (jitendar-singh) [reply smoke/features/jenkins-ephemeral.feature:16]: All the step definitions for the above are already present so didnt create a new one, as we use the same before starting the persistent template test

**Pull request merged**